packages/api/src/decorator/private/activityDecoratorRequest.ts

```diff
@@ -1,6 +1,23 @@
 type ActivityDecoratorRequestType = {
+  /**
+   * Decorate the activity as it participate in a livestreaming session.
+   *
+   * - `"completing"` - decorate as the livestreaming is completing
+   * - `"ongoing"` - decorate as the livestreaming is ongoing
+   * - `"preparing"` - decorate as the livestreaming is being prepared
+   * - `undefined` - not participated in a livestreaming session
+   */
+  decorateForLivestreaming: 'completing' | 'ongoing' | 'preparing' | undefined;
+
```

Comment:
How we would add a new type here? Imagine we'd like to decorate for reaction. Would it be a new `decorateForReaction` prop?

Most of fields are named the way they describe a value compared to the new one describing the action to be done (decorate something). Not against the naming, but it'd be nice to have an insight on why we introduce such.

Reply:
In our offline chat, I thought you want to name it `decorateForXXX`. I feel weird too. 🤣

I guess we would name it `livestreaming` or `livestreamingState`. What's your thought?

Reply:
Sorry for confusion, I'm ok with both specific field names proposed. Alternatively we could go with an array of decorations (`decoration: []` field) maybe.

Reply:
Sounds good!